README.md

```diff
@@ -11,55 +11,62 @@ A data visualization curriculum of interactive notebooks, using [Vega-Lite](http
      [Jupyter Book](https://uwdata.github.io/visualization-curriculum/altair_introduction.html) |
      [Jupyter Notebook](https://github.com/uwdata/visualization-curriculum/blob/master/altair_introduction.ipynb) |
      [Open in Colab](https://colab.research.google.com/github/uwdata/visualization-curriculum/blob/master/altair_introduction.ipynb) |
-     [Open in Observable](https://observablehq.com/@uwdata/introduction-to-vega-lite)
+     [Open in Observable](https://observablehq.com/@uwdata/introduction-to-vega-lite) |
+     [Open in Nextjournal](https://github.nextjournal.com/uwdata/visualization-curriculum/blob/master/altair_introduction.ipynb)
    </small>
 
 2. Data Types, Graphical Marks, and Visual Encoding Channels<br/>
    <small>
      [Jupyter Book](https://uwdata.github.io/visualization-curriculum/altair_marks_encoding.html) |
      [Jupyter Notebook](https://github.com/uwdata/visualization-curriculum/blob/master/altair_marks_encoding.ipynb) |
      [Open in Colab](https://colab.research.google.com/github/uwdata/visualization-curriculum/blob/master/altair_marks_encoding.ipynb) |
-     [Open in Observable](https://observablehq.com/@uwdata/data-types-graphical-marks-and-visual-encoding-channels)
+     [Open in Observable](https://observablehq.com/@uwdata/data-types-graphical-marks-and-visual-encoding-channels) |
+     [Open in Nextjournal](https://github.nextjournal.com/uwdata/visualization-curriculum/blob/master/altair_marks_encoding.ipynb)
```

Comment:
Nextjournal should probably come before observable since observable is js and the rest are python. Can you make that a bit clearer (e.g. add **Python** or **JavaScript** to the description somewhere)?